lib/command.ts

```diff
@@ -332,3 +331,26 @@ Command.setReplyTransformer('hgetall', function (result) {
   }
   return result
 })
+
+class MixedBuffers {
+  length = 0
+  items = []
+
+  public push(x: string | Buffer) {
+    this.length += Buffer.byteLength(x);
+    this.items.push(x)
+  }
+
+  public toBuffer(): Buffer {
+    const result = Buffer.alloc(this.length);
```

Comment:
I believe `Buffer.allocUnsafe` would be better for perf here

Reply:
replaced w/ Buffer.allocUnsafe